src/Misc/Utils.cpp

```diff
@@ -15,15 +15,16 @@ int Abs(int x)
 
 std::string ShortenString(const std::string& str, size_t maxLength)
 {
+    if (str.size() <= maxLength) return str;
     std::string out = str;
     out.erase(out.begin() + maxLength, out.end());
     // Don't add an ellipsis if the string is very short
-    if (maxLength > 5)
+    if (maxLength > 5 && str.size() > 5)
```

Comment:
```suggestion
    if (maxLength > 5)
```
Line 18 already ensures that `str.size() > maxLength`, so this check implicitly requires that `str.size() > 5`